### DISCUSSION TITLE
Ability to go between automations

### DISCUSSION THREAD

**Discussion by valgeirhelgi** [home-assistant/frontend#8237] [Configuration]
Currently if the user would like to edit a couple of automations in a row, i.e. change their name, the user would have to do the following steps.
1. Enter an automation
2. Edit automation
3. Save edit
4. Go back to the automation dashboard
5. Enter the next automation
6. etc.

I'm suggesting that instead it would be possible to choose the next or previous automation, hence skip steps four and five here above, so the user is able to choose previous and next at the bottom of the automation, maybe next to the save button.

This feature could also be implemented in devices, entities, areas, scenes, scripts and so on.

**Comment by spacegaier** [comment]
I am not sure if that is really needed. How often do you have to mass rename automations? What is the use case here?

**Reply by valgeirhelgi** [reply]
It's a question of ease of use. For example when creating your first automations and you want to double check if the automations are setup correctly, same applies to the other things like enities and more. When you have finished onboarding and you are setting all the devices to the their correct area, pressing next or previous after sorting by area for example, and start working on changing to correct areas from no area.

Basically simple use across the configuration part of home assistant.
